common/djangoapps/track/backends/tests/test_oli.py

```diff
@@ -0,0 +1,201 @@
+"""
+Tests oli.py
+"""
+from __future__ import absolute_import
+
+from mock import patch
+import logging
+from django.test import TestCase
+from south.utils import datetime_utils as datetime
+from track.backends.oli import OLIAnalyticsBackend
+from student.tests.factories import UserFactory
+log = logging.getLogger(__name__)
+
+
+# pylint: disable=line-too-long
```

Comment:
In all 3 test cases, the json string for the event object caused line-too-long error.  If it's better to disable at each individual method level, I can do that, instead of at the class level.